students/sangwoong/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from users.views import SignUpView
+
+urlpatterns = [
+  path("/sign-up", SignUpView.as_view())
```

Comment:
signup은 통상적으로 붙여서 사용합니다.
```suggestion
  path("/signup", SignUpView.as_view())
```